demo/admin/src/products/categories/AssignedProductsTab.tsx

```diff
@@ -0,0 +1,57 @@
+import { CancelButton, SaveBoundary, SaveBoundarySaveButton } from "@comet/admin";
+import { Add as AddIcon } from "@comet/admin-icons";
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from "@mui/material";
+import { AssignedProductsGrid } from "@src/products/categories/AssignedProductsGrid";
+import { AssignProductsGrid } from "@src/products/categories/AssignProductsGrid";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+type Props = {
+    productCategoryId: string;
+};
+
+export function AssignedProductsTab({ productCategoryId }: Props): React.ReactElement {
+    const [isOpen, setIsOpen] = React.useState(false);
+
+    const handleCloseDialog = () => {
+        setIsOpen(false);
+    };
+    return (
+        <>
+            <AssignedProductsGrid
+                toolbarAction={
+                    <Button startIcon={<AddIcon />} onClick={() => setIsOpen(true)} variant="contained" color="primary">
+                        <FormattedMessage id="products.editProductCategory.assignProducts" defaultMessage="Assign Products" />
+                    </Button>
+                }
+                filter={{ category: { equal: productCategoryId } }}
+            />
+            <SaveBoundary
+                subRoutePath="./save-this"
+                onAfterSave={() => {
+                    setIsOpen(false);
+                }}
+            >
+                <Dialog open={isOpen} onClose={handleCloseDialog} fullWidth maxWidth="xl">
+                    <DialogTitle>
+                        <FormattedMessage id="products.editProductCategory.assignProducts" defaultMessage="Assign Products" />
+                    </DialogTitle>
+                    <DialogContent
+                        sx={{
+                            height: "70vh",
+                            padding: 0,
+                            paddingTop: "0 !important" /* is connected to title-style */,
```

Comment:
It also adds the dialogs background-color, I'd keep `DialogContent` here, even if we don't want the padding.

Reply:
I believe this would be the "correct" way to set the top padding 😅 

```suggestion

                            [`${dialogTitleClasses.root} + &`]: {
                                paddingTop: 0,
                            },
```

Reply:
@jamesricky this does not work/is not enough. 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/eeb693c4-c925-46db-baa8-c196ca48270c">

Reply:
and should this really be needed? it looks like a style-bug hack.

Reply:
I just took a look at this locally. 
Since the DataGrid fills out the entire Dialog anyway, there is no need for `DialogContent`. 
I think a `Box` with `height: 70vh` should be enough. 

```tsx
<Box height="70vh">
    <AssignProductsGrid productCategoryId={productCategoryId} />
</Box>
```